__init__.py

```diff
@@ -17,34 +17,32 @@
 
 import gettext
 import html
-import json
 import os
 import time
+
+from pyradios import RadioBrowser
 from shutil import which
 from signal import SIGTERM
 from subprocess import Popen
-from urllib.request import Request, urlopen
-from urllib.error import URLError
 
 from pext_base import ModuleBase
 from pext_helpers import Action, SelectionType
 
 
 class Module(ModuleBase):
-    def init(self, settings, q):
+    def __init__(self, settings, q):
         self.module_path = os.path.dirname(os.path.abspath(__file__))
 
         try:
-            lang = gettext.translation('pext_module_radio', localedir=os.path.join(self.module_path, 'locale'), languages=[settings['_locale']])
+            lang = gettext.translation('pext_module_radio', localedir=os.path.join(self.module_path, 'locale'),
+                                       languages=[settings['_locale']])
         except FileNotFoundError:
             lang = gettext.NullTranslations()
             print("No {} translation available for pext_module_radio".format(settings['_locale']))
 
         lang.install()
 
-        self.baseUrl = 'http://www.radio-browser.info/webservice' if ('baseurl' not in settings) else settings['baseurl']
-
-        self.useragent = 'Pext RadioBrowser/Development' if ('useragent' not in settings) else settings['useragent']
```

Comment:
Seeing how you're removing the useragent and baseurl settings, you should probably also remove them in generate_metadata.py (https://github.com/Pext/pext_module_radio/blob/master/generate_metadata.py#L49-L56) and run generate_metadata.py to generate new metadata files in all languages without these options)